lib/ObjWriter/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ endif()
 add_library(objwriter
    SHARED
    objwriter.cpp
+   objwriter.h
```

Comment:
Please remove headers from the set of C++ files to compile; they should not be needed if the include paths are configured properly.